htlcswitch/interfaces.go

```diff
@@ -41,6 +41,23 @@ type InvoiceDatabase interface {
 	HodlUnsubscribeAll(subscriber chan<- interface{})
 }
 
+// packetHandler is an interface used exclusively by the Switch to handle
+// htlcPacket and pass them to the link implementation.
+type packetHandler interface {
+	// handleSwitchPacket handles the switch packets. This packets might be
+	// forwarded to us from another channel link in case the htlc update
+	// came from another peer or if the update was created by user
+	// initially.
+	//
+	// NOTE: This function MUST be non-blocking (or block as little as
```

Comment:
nit: Though it's copied, I think the comments contradict themselves...

Reply:
Changed the comment, PTAL